### ISSUE TITLE
Correct Hyper Parameters for the "Original", "Retrain", "EUK"

### ISSUE THREAD

**Issue by Ldoun** [meghdadk/SCRUB#4]
First of all, Thanks for the good work.
While trying out the code I found that some of the hyperparameters are not listed in the paper or the hyperparameters detailed in the paper are not used in the experiments.

The hyperparameters for the "Original" and "Retrain" were not listed in the paper(or I couldn't find them). And for the detailed hyperparameters of "EUK" are mentioned as "same setting as Retrain baseline". Based on the main_merged.py I assumed that detailed hyperparameters of "Retrain" are [epochs 26, fixed learning rate 0.01, weight_decay 5e-4, momentum 0.9]. But In the MIA_experiments.ipynb the "EUK" baseline uses learning rate decay every 10, 15, and 20 epochs and uses 0 for the optimizer's weight_decay and the momentum.

lastly, "Finetune", "Negrad", "CFK", and "EUK" should use weight_decay of 5e-4 but it is implemented to use 0.

This got me confused. Can you provide the exact value for the hyperparameters? (including epochs, learning rate, weight_decay, momentum, lr decay) 

Thank You! If I missed something let me know
